website/markdown/docs/contribution/getting-started.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-name: Getting started
```

Comment:
I'd change the name of the file too and the reference from `links.mdx` and probably some other places.

Reply:
I was afraid of changing the name of the file because it's referenced for a lot of places, including github and old articles

Reply:
Ok, I've done it and set a redirect from `getting-started` to `get-started` for links that our out of our control